src/themes/elife/index.ts

```diff
@@ -22,6 +23,16 @@ const getArticleId = (): string => {
   return first(selector)?.innerHTML ?? ''
 }
 
+const getArticleDoi = (): string => {
+  const selector =
+    ':--identifier meta[content="https://registry.identifiers.org/registry/doi"] ~ [itemprop="value"]'
+  return first(selector)?.innerHTML ?? ''
+}
+
+const getArticleTitle = (): string => {
+  return first(':--title')?.innerHTML ?? ''
```

Comment:
`innerHTML` can include line breaks, which get encoded into the title string sent to social media. Thema's `text` util might be a better choice here.

Reply:
Addressed in 483bcbf.